File: src/main/kotlin/com/nuutrai/reactor/tick/Ticker.java
package com.nuutrai.reactor.tick;

import com.nuutrai.reactor.data.DataManager;
import org.bukkit.Bukkit;
import org.bukkit.entity.Player;

public class Ticker {
    public void tick() {
        for (Player player : Bukkit.getOnlinePlayers()) {
            DataManager.get(player).tick();
        }
    }
}

File: src/main/kotlin/com/nuutrai/reactor/tick/Ticker.kt
package com.nuutrai.reactor.tick

import com.nuutrai.reactor.data.DataManager
import org.bukkit.Bukkit

class Ticker {
	fun tick() {
		for (player in Bukkit.getOnlinePlayers()) {
			DataManager.get(player)?.tick() ?: continue
		}
	}
}
